src/test/kotlin/com/terraformation/backend/nursery/NurserySearchTest.kt

```diff
@@ -272,7 +274,8 @@ internal class NurserySearchTest : DatabaseTest(), RunsAsUser {
                       "notReadyQuantity" to "16",
                       "readyQuantity" to "32",
                       "totalQuantity" to "${16 + 32}",
-                      "totalQuantityWithdrawn" to "${8192 + 16384 + 65536 + 131072}",
+                      "totalQuantityWithdrawn" to
+                          numberFormat.format(8192 + 16384 + 65536 + 131072),
```

Comment:
Would it be better to put in the literal string here for the test? Using `numberFormat.format` feels a little circular.

Reply:
The reason I didn't put the literal string in there is that it would make it unclear where the value came from: `8192 + 16384 + 65536 + 131072` is clearly the sum of some specific numbers, but `"221,184"` looks arbitrary and I'd need to add a comment to explain it at which point I felt like it'd be more verbose than calling `numberFormat.format()`.

All that said, I can make the intent clearer here; will update it.

Reply:
Cool, thanks!